Rock.Messaging.RabbitMQ/RabbitEngine.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Threading.Tasks;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rock.Messaging.RabbitMQ
+{
+    public class RabbitEngine : IReceiver, ISender
+    {
+        private readonly IRabbitSessionConfiguration config;
+        private ConnectionFactory connectionFactory;
+        private IConnection connection;
+        private List<IModel> consumerModels; // While this is hacky- IModel is not thread safe.. see: https://www.rabbitmq.com/releases/rabbitmq-dotnet-client/v1.7.1/rabbitmq-dotnet-client-1.7.1-user-guide.pdf
```

Comment:
I believe I forgot to add the IModel created in the Receiver.Start() method to the list.

The objects appear to clean up when an application isn't using them any longer, but the research I did when I was writing this wrapper indicated that IModels should be disposed. 

Good catch!